setup.py

```diff
@@ -34,11 +34,17 @@
         "License :: OSI Approved :: MIT License",
     ],
     keywords=["x25519", "gimli", "signatures"],
-    package_dir={"lithium": "py_src"},
+    package_dir={"lithium": "py"},
     packages=["lithium"],
     include_package_data=True,
     package_data={"build_tools": ["ffibuilder.py"]},
     install_requires=["cffi>=1.12.0"],
     setup_requires=["cffi>=1.12.0", "setuptools_scm"],
     cffi_modules=["ffibuilder.py:ffibuilder"],
+    scripts=[
```

Comment:
nit: I think we should prefer to use `entry_points`; see discussion here: https://stackoverflow.com/questions/18787036/difference-between-entry-points-console-scripts-and-scripts-in-setup-py

Reply:
done